src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -605,6 +605,10 @@ public function testChgrpSymlink()
      */
     public function testChgrpSymlinkFails()
     {
+        if (defined('HHVM_VERSION')) {
+            $this->markTestSkipped('Shouldn\'t be executed in HHVM context.');
```

Comment:
This PR already fixes the test in the Security componentes. Or you have any comment about the change in the Security component tests?